core/app/models/spree/zone.rb

```diff
@@ -42,13 +41,17 @@ class Zone < Spree::Base
     self.whitelisted_ransackable_attributes = ['description']
 
     def self.default_tax
-      where(default_tax: true).first
+      Spree::Deprecation.warn("There is no default tax zone anymore.", caller)
+      all.detect { |zone| zone.try(:default_tax) }
```

Comment:
If the column is gone, there is no need for this anymore, right? Maybe we should keep the column for at least one minor version instead?